src/main/java/Event.java

```diff
@@ -0,0 +1,21 @@
+public class Event extends Todo {
+    protected String at;
```

Comment:
The naming "at" might be a little be confusing. It might be better if you can mention something like eventTime.